frontend/src/metabase/query_builder/components/expressions/analytics.ts

```diff
@@ -0,0 +1,10 @@
+import { trackSchemaEvent } from "metabase/lib/analytics";
+import * as Lib from "metabase-lib";
+
+export const trackColumnCombineViaShortcut = (query: Lib.Query) => {
```

Comment:
Make sure to add e2e tests for this event. Search for `describeWithSnowplow` for examples.